melvs/melvsboot/src/main/java/com/melvs/melvsboot/Controller.java

```diff
@@ -0,0 +1,56 @@
+package com.melvs.melvsboot;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+//controller rest mapping
+@RestController
+public class Controller {
+
+//means needs another dependencies to map
+    @Autowired
+    private BusinessService bs;
+
+//request mapping endpoint
+    @RequestMapping("/")
+    public String index(){
+        return "This is a simple test SpringBoot";
+    }
+
+    @RequestMapping("/GetEnd")
+    public List<GetList> getAllList() {
+        return bs.getAllList();
+        }
+
+//for POST request
+    @RequestMapping(method = RequestMethod.POST, value ="/PostEnd")
```

Comment:
Again. Interesting implementation of Mapping here.. You simply used @RequestMapping provided it with parameters. The way I implemented mine was by importing @PostMapping, @PutMapping, etc.. which gave me a lot of imports at the top of my code. I am not sure what difference using @RequestMapping vs @PostMapping, but your code got more concise because of it.

I also used @Path annotation to specify the path of my REST methods, while here it's just a parameter. Very interesting.

Reply:
yes this is a @RequestMapping which map a HTTP method, added link of tutorial in our shared docs

Reply:
I only use RequestMapping on a class level. I use [Http Method]+Mapping annotations so I don't need to define request method. GetMapping, PostMapping, PutMapping and so on...